### DISCUSSION TITLE
[Feature] Albums - option to show only non-album photos

### DISCUSSION THREAD

**Discussion by greentroll88** [immich-app/immich#7519] [Feature Request]
### The feature

Right now, you can't view only photos that are not included in any of the albums in the mobile app or web interface. Because of this, sorting photos into albums is not very convenient - you have to scroll through all the photos instead of viewing only the unsorted ones and distributing them into albums.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can do this via the search
